tests/unit/select_connection_ioloop_tests.py

```diff
@@ -10,22 +10,33 @@
 
 import logging
 try:
+    #pylint: disable=F0401
```

Comment:
Need to put all these pylint pragmas on the same line as the imports, so they don't disable checks on all the other imports